Anima/math-tools/multi_compartment_base/animaMCMConstants.h

```diff
@@ -28,4 +29,58 @@ const double DiffusionSmallDelta = 10.0e-3;
 //! Default big delta value (classical values)
 const double DiffusionBigDelta = 40.0e-3;
 
+//! Default zero lower bound (in case we want something else than zero one day)
+const double MCMZeroLowerBound = 0.0;
+
+//! Epsilon value in case we do not want to have parameters reaching their true bounds (used for now in DDI)
+const double MCMEpsilon = 1.0e-2;
+
+//! Axial diffusivity add on to lower bound (used to ensure a minimal anisotropy to the anisotropic compartments)
+const double MCMAxialDiffusivityAddonLowerBound = 5.0e-4;
+
+//! Diffusivity lower bound for estimation
+const double MCMDiffusivityLowerBound = 1.0e-5;
+
+//! Diffusivity upper bound
+const double MCMDiffusivityUpperBound = 3.0e-3;
+
+//! Radial diffusivity upper bound
+const double MCMRadialDiffusivityUpperBound = 1.0e-3;
+
+//! Free water diffusivity lower bound
+const double MCMFreeWaterDiffusivityLowerBound = 2.0e-3;
+
+//! Free water diffusivity upper bound
+const double MCMFreeWaterDiffusivityUpperBound = 4.0e-3;
+
+//! Isotropic restricted water diffusivity lower bound
+const double MCMIsotropicRestrictedWaterDiffusivityLowerBound = 0.25e-4;
+
+//! Isotropic restricted water diffusivity upper bound
+const double MCMIsotropicRestrictedWaterDiffusivityUpperBound = 2.0e-3;
+
+//! Stanisz water diffusivity lower bound
```

Comment:
These are not necessary as completely redundant with DiffusivityBounds, i.e. axial diffusivity bounds.

Reply:
Well you have set yourself the lower bound to 1.5e-4 in a previous commit, plus I don't want Stanisz to take over the free water compartment. So that's why I kept them

Reply:
Stanisz won't take over as there is also the radius.